pkg/server/restart.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/containerd/containerd"
 	"github.com/containerd/containerd/content"
 	"github.com/containerd/containerd/errdefs"
+	imagemanage "github.com/containerd/containerd/images"
```

Comment:
This name is confusing. I prefer `containerdimages` if we really need another name for the package.

Reply:
done